media/juxtapose/css/playhead.css

```diff
@@ -4,6 +4,9 @@
 .jux-playhead-container input[type=range] {
     cursor: col-resize;
     width: 930px;
+    -webkit-appearance: none;
```

Comment:
Just a note - this doesn't seem to be working in Firefox, even with `-moz-appearance: none`